### PR TITLE
Create Web_Template_NNN

### PR DESCRIPTION
**Pull request by harry97sharma**
I think my project would be a great fit for up-for-grabs.

### CODE REVIEW

**Review by harry97sharma — COMMENTED** [review]
Forgot to add extension in file

File: _data/projects/WT_nnn.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: WEB_TEMPLATE_001
+desc: Cool Colurful Template for easy web development
+site: https://example.org/your-project-link-here
+tags:
+  - html
+  - web-development
+  - web-templates
+upforgrabs:
+  name: help wanted by a beginner
+  link: https://github.com/harry97sharma/WEB_TEMPLATE_001/issues?q=label%3A%22help+wanted%22+label%3A%22good+first+issue%22
```
Comment (shiftkey) [inline _data/projects/WT_nnn.yml:10]:
```suggestion
  name: good first issue
  link: https://github.com/harry97sharma/WEB_TEMPLATE_001/labels/good%20first%20issue
```

File: _data/projects/WT_nnn.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: WEB_TEMPLATE_001
+desc: Cool Colurful Template for easy web development
+site: https://example.org/your-project-link-here
```
Comment (shiftkey) [inline _data/projects/WT_nnn.yml:3]: :art: this needs to be a real site

File: _data/projects/WT_nnn.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: WEB_TEMPLATE_001
+desc: Cool Colurful Template for easy web development
```
Comment (shiftkey) [inline _data/projects/WT_nnn.yml:2]: Please provide a good description here of this project and what contributions you are interested in

File: _data/projects/WT_nnn.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+name: WEB_TEMPLATE_001
```
Comment (shiftkey) [inline _data/projects/WT_nnn.yml:1]: This repository is very new, and I'm not sure it's a good fit for being listed here based on the details provided so far

**Review by shiftkey — CHANGES_REQUESTED** [review]

**Comment by shiftkey** [comment]
Closing this out due to inactivity. Please re-submit if you are still interested and are able to address the feedback above.

**Pull request closed without merging**